demos/llama_chat/python/client.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import argparse
+import os
+
+from ovmsclient import make_grpc_client
+from transformers import AutoTokenizer
+
+parser = argparse.ArgumentParser(description='Inference script for generating text with llama')
+
+parser.add_argument('--url', required=True, type=str)
+parser.add_argument('--question', required=True, type=str)
+parser.add_argument('--seed', required=False, type=int, default=0)
+parser.add_argument('--actor', required=False, type=str, choices=['general-knowledge', 'python-programmer'], default='general-knowledge')
+args = parser.parse_args()
+
+np.random.seed(args.seed)
+client = make_grpc_client(args.url)
+
+GENERAL_PRE_PROMPT = """You are super-human who is able to answer any question. All questions and answers should end with [EOS].
+Question: What is the capital of Poland? [EOS]
+Answer: Warsaw. [EOS]
+
+Question: <Q> [EOS]
+Answer:"""
+
+PROGRAMMER_PRE_PROMPT = """You are super-programmer who is brilliant python coder. All the answers contain code snippets ending with [EOS].
+Question: Write short python function to sum numbers in a list. [EOS]
```

Comment:
Do we need to post example question/answer each time?

Reply:
This script does not support chatting, only answers given question one time